presentational/Navbar.js

```diff
@@ -1,7 +1,102 @@
+import Link from 'next/link'
+import React from 'react'
+import { withRouter } from 'next/router'
 import styled from 'styled-components'
 
-const Navbar = styled.div`
-  border: 1px solid black;
+const NavWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  border: 2px;
+
+  justify-content: center;
+  align-items: center;
+`
+
+const LogInWrapper = styled.div`
+  padding-right: 30px;
+  display: flex;
+  position: absolute;
+  top: 0;
+  right: 0;
+  font-family: ${({ theme }) => theme.fonts.text};
+  font-size: 20px;
+  font-weight: 600;
+  height: 100%;
+  align-items: center;
+`
+
+const TitleWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-right: 20px;
+  margin-left: 20px;
+`
+
+const StyledLink = styled.a`
+  text-decoration: none;
+  opacity: ${({ active, theme }) =>
+    active ? 1 : theme.transparencies.inactive};
+  margin: 20px;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
 `
+const PrimaryTitle = styled.span`
+  color: ${({ theme }) => theme.colors.primary};
+  font-size: 30px;
+  font-family: ${({ theme }) => theme.fonts.title};
+`
+const PrimaryMutedTitle = styled.span`
+  font-size: 30px;
+  color: ${({ theme }) => theme.colors.primaryMuted};
+  font-family: ${({ theme }) => theme.fonts.title};
+`
```

Comment:
These are the same except for color. Could it make sense to have muted be a prop?
```suggestion
const Title = styled.span`
  font-size: 30px;
  font-family: ${({ theme }) => theme.fonts.title};
  color: ${({ theme, muted }) =>
    muted ? theme.colors.primaryMuted : theme.colors.primary};
`
```